new release 0.10.18 (#492)
pkgcore/pkgcheck: NEWS.rst

Release Notes
=============

----------------------------
pkgcheck 0.10.18 (2022-11-09)
----------------------------

> Special thanks is given to Sam James, for continues support during all

Reviewer: That's so kind of you! Thanks so much Arthur! 😊

development, bringing ideas, testing and improving, and especially proofreading
and improving all docs, texts and help messages. Every release is better thanks
to him.

> - Network checks: fix wrong attributes shown when same URL is checked (#403,

Reviewer: ```suggestion
- Network checks: fix wrong attributes ("blame") shown when same URL is checked (#403,
```

  Arthur Zamarin)

- BetterCompressionCheck: new check for suggesting better compression URI for

>   distfile fetch (#483, Arthur Zamarin)

Reviewer: ```suggestion
  gitlab and github distfiles (#483, Arthur Zamarin)
```

- ExcessiveLineLength: report lines longer than 120 characters (with multiple
  exception rules) (#480, Arthur Zamarin)

> - MissingRemoteIdCheck: new check for suggesting missing remote ids, inferred

Reviewer: I think:
```suggestion
- MissingRemoteIdCheck: new check for suggesting missing remote-ids, inferred
```

This is kind of a nit though and what you did is technically fine, it's just that in a Gentoo context, they're _also_ "remote-id" as that's the name in XML.

  from HOMEPAGE and SRC_URI (#486, Arthur Zamarin)

> - DoCompressedFilesCheck: new check for calling ``doman``, ``newman`` and

Reviewer: Oxford comma:
```suggestion
- DoCompressedFilesCheck: new check for calling ``doman``, ``newman``, and
```

  ``doinfo`` with compressed files (#485, Arthur Zamarin)

> - AcctCheck: use ``metadata/qa-policy.conf`` for dynamic id range (#489, Arthur

Reviewer: ```suggestion
- AcctCheck: determine dynamic ID range from repository file ``metadata/qa-policy.conf`` rather than static hardcoded values in pkgcheck (#489, Arthur
```
